<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link href="static/bootstrap.min.css" rel="stylesheet">
    <link href="static/style.css" rel="stylesheet">

    <title>Engineering2024_Northwest_Union</title>

</head>
<body>
  <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
  <div class="container">

    <!---- TEAM NAME ---->
    <a class="navbar-brand" href="#">Northwest-Union</a>

    <!---- SMALL SCREEN MENU ICON ---->
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto left-aligned">

        <!---- HOME ---->
        <li class="nav-item">
          <a class="nav-link" href="index.html">Home</a>
        </li>

        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Project
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="description.html">Description</a></li>
            <li><a class="dropdown-item" href="contribution.html">Contribution</a></li>
            <li><a class="dropdown-item" href="parts.html">Parts</a></li>
          </ul>
        </li>

        <!---- LAB ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Wet Lab
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="engineering.html">Engineering</a></li>

            <li> <a class="dropdown-item" href="results.html">Results</a></li>

            <li><a class="dropdown-item" href="hardware.html">Hardware</a></li>
            <li> <a class="dropdown-item" href="protocol.html">Protocol</a></li>
            <li> <a class="dropdown-item" href="notebook.html">Notebook</a></li>

          </ul>
        </li>

        <!---- SAFETY ---->
        <li class="nav-item">
          <a class="nav-link" href="safety.html">Safety</a>
        </li>

        <!---- HUMAN PRACTICES ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Human Practices
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="human-practices.html">Integrated Human practices</a></li>
            <li> <a class="dropdown-item" href="education.html">Education</a></li>
            <li> <a class="dropdown-item" href="sustainable.html">Sustainable Development Impact</a></li>
            <li> <a class="dropdown-item" href="entrepreneurship.html">Entrepreneurship</a></li>
            <li> <a class="dropdown-item" href="collaboration.html">Collaboration</a></li>
          </ul>
        </li>

        <!---- TEAM ---->
        <li class="nav-item">
          <a class="nav-link" href="members.html">Team</a>
        </li>
      </ul>
    </div>
  </div>
</nav>

  <!-- Header -->
  <header class="bg-hero py-5 mb-5">
    <div class="container h-100">
      <div class="row h-100 align-items-center">
        <div class="col-lg-12">
          <h1 class="display-4 text-white mt-5 mb-2">Engineering</h1>
	        <p class="lead mb-5 text-white-50"></p>
        </div>
      </div>
    </div>
  </header>

  <!-- Page Content -->
  <div class="container">
    

<div class="row mt-4">
  <div class="col">
    <h2>1.1. First Iteration - Chassis Engineering</h2>
    <hr>
    <h3>Design</h3>
    <p>Considering the need to monitor blood glucose levels and regulate glucose and lipid metabolism through the released short-chain fatty acids (SCFAs) and glucagon-like peptide-1 (GLP-1), we selected the probiotic strain Escherichia coli Nissle 1917 (ECN1917) as the chassis organism for the entire system. Additionally, since we need to test the engineered bacteria under various conditions, it is essential to assess the compatibility of the chassis organism with all experimental conditions.</p>
    <h3>Build</h3>
    <p>To simulate different blood glucose concentrations, we designed L.B. agar plates with varying glucose levels: L.B. agar plates without glucose, L.B. agar plates with 2.0 mmol/L glucose, L.B. agar plates with 5.0 mmol/L glucose, L.B. agar plates with 8.0 mmol/L glucose, and L.B. agar plates with 11.0 mmol/L glucose, to represent low, normal, and high blood glucose ranges. Subsequently, we inoculated these agar plates with the same dilution factor of the chassis ECN1917 as well as the engineered bacteria containing the introduced plasmid to evaluate their growth status.</p>
    <h3>Test</h3>
    <p>The results indicated that all types of bacteria grew normally on the various plates, and there were no statistically significant differences in colony counts.</p>
    <h3>Study</h3>
    <p>Based on the above findings, we proceeded to the next iteration, which involved encapsulating the engineered bacteria within vesicles.</p>
  </div>
</div>

<div class="row mt-4">
  <div class="col">
    <h2>1.2. Second Iteration - Chassis Engineering</h2>
    <hr>
    <h3>Design</h3>
    <p>In this iteration, we designed self-assembling nanovesicles based on 1,2-distearoyl-sn-glycero-3-phosphoethanolamine (DLPC) to prevent the potential exposure of immunogenic components and the division of engineered bacteria. 1,2-distearoyl-sn-glycero-3-phosphoethanolamine-polyethylene glycol (DSPE-PEG) served as the primary component of the membrane, while cholesterol (1% w/w) was added to enhance membrane stability. Calcium ion solution was used to promote the self-assembly of the membrane on the surface of the engineered bacteria.</p>
    <h3>Build</h3>
    <p>We diluted and inoculated both the conventional engineered bacteria and the engineered bacteria loaded in nanovesicles onto the agar plates used in the first iteration to investigate whether the nanovesicles inhibited bacterial division. Additionally, we employed DMAO/PI staining to determine whether the engineered bacteria maintained normal viability.
    </p>
    <h3>Test</h3>
    <p>We found that the engineered bacteria not encapsulated in nanovesicles formed normal colonies on the plates, while those encapsulated in nanovesicles formed very few colonies. This indicates that the nanovesicles indeed inhibited bacterial division. The fluorescent staining results for the engineered bacteria demonstrated that, regardless of encapsulation in nanovesicles, the engineered bacteria exhibited viability.</p>
    
    <h3>Study</h3>
    <p>Based on the aforementioned results, we proceeded to the next iteration aimed at determining whether the substances produced by the engineered bacteria could be released into the external environment of the nanovesicles.

    </p>
  </div>
</div>

<div class="row mt-4">
  <div class="col">
    <h2>1.3. Third Iteration - Chassis Engineering</h2>
    <hr>
    <h3>Design</h3>
    <p>In this iteration, we validated the ability of the engineered bacteria encapsulated in nanovesicles to secrete target products, comparing them with normal engineered bacteria.

    </p>
    <h3>Build</h3>
    <p>We cultured the same concentration of normal engineered bacteria and those encapsulated in nanovesicles in PBS buffer with the highest glucose concentration (11.0 mmol/L), designed to inhibit the proliferation of normal engineered bacteria due to the lack of exogenous amino acids and nucleotides. To detect the secretion of SCFAs and GLP-1 recombinant protein, we employed high-performance liquid chromatography (HPLC) and an ELISA kit for GLP-1 to measure the levels of target substances in the buffer.</p>

    <h3>Test</h3>
    <p>The results indicated that both normal engineered bacteria and those encapsulated in nanovesicles were able to produce the target products. However, the GLP-1 yield was 53.7% of that from the normal bacteria. This suggests that although the GLP-1 recombinant protein can cross the phospholipid composite vesicles, its diffusion rate per unit time is lower. </p>
    <h3>Study</h3>
    <p>Based on the aforementioned results, we will further investigate the expression of the target protein in the next steps.

    </p>
  </div>
</div>

<div class="row mt-4">
  <div class="col">
    <h2>2.1. First Iteration - Gene Expression Verification Engineering</h2>
    <hr>
    <h3>Design</h3>
    <p>In this iteration, we need to first verify the successful transformation of the recombinant plasmid into the chassis bacteria.</p>

    <h3>Build</h3>
    <p>Since the recombinant plasmid contains a kanamycin resistance gene, we will dilute the bacterial suspension using an electroporator and inoculate it onto kanamycin L.B. agar plates to select for resistant colonies. After that, we will pick single colonies for amplification, extract total RNA according to the requirements of the total RNA extraction kit, and perform real-time quantitative PCR using designed primers to verify whether the recombinant plasmid has been successfully introduced into the engineered bacteria.</p>

    <h3>Test</h3>
    <p>The results indicate that the chassis bacteria without the plasmid showed no growth on the kanamycin plates, while the engineered bacteria formed colonies on the kanamycin plates. PCR results confirm that the plasmid was successfully introduced into the engineered bacteria.</p>
  <h3>Study</h3>
  <p>Based on the above experimental results, we conducted further validation of protein expression.

  </p>
  
  </div>
</div>

<div class="row mt-4">
  <div class="col">
    <h2>2.2. Second Iteration - Gene Expression Verification Engineering</h2>
    <hr>
    <h3>Design</h3>
    <p>First, we will verify the expression of the key protein, butyryl-CoA: acetate CoA-transferase (BCoAT), responsible for producing SCFAs in the engineered bacteria.</p>
    <h3>Build</h3>
    <p>According to existing literature and databases, we estimate that the molecular weight of the BCoAT protein transcribed from the pET28a-BCoAT-His plasmid should be around 46 kDa. We cultured the engineered bacteria in L.B. liquid medium with the highest glucose concentration for 8 hours to reach the logarithmic growth phase, then lysed the bacteria on ice using an ultrasonic cell disruptor and centrifuged at high speed to obtain the cell-free products. Protein separation was performed using SDS-PAGE gel electrophoresis, and specific detection was carried out using His antibodies.</p>

    <h3>Test</h3>
    <p>We have successfully isolated the BCoAT protein, and its molecular weight is consistent with the expected value.</p>

    <h3>Study</h3>
    <p>Based on the above findings, we will further verify the expression of the second target protein.</p>


  </div>
</div>

<div class="row mt-4">
  <div class="col">
    <h2>2.3. Third Iteration - Gene Expression Verification Engineering</h2>
    <hr>
    <h3>Design</h3>
    <p>Throughout the project, we used two strategies—enzyme-linked immunosorbent assay (ELISA) and SDS-PAGE gel electrophoresis—to detect the expression of GLP-1 recombinant protein. In this section, we infer that the molecular weight of the GLP-1 recombinant protein is between 74-76 kDa.

    </p>

    <h3>Build</h3>
    <p>We cultured the engineered bacteria in L.B. liquid medium with the highest glucose concentration for 8 hours to reach the logarithmic growth phase, then performed high-speed centrifugation to collect the supernatant. Protein concentration was determined using a BCA protein quantification kit, and proteins were separated by SDS-PAGE. Subsequently, we used FLAG tag antibodies for specific Western blot screening to capture the target protein.</p>

    <h3>Test</h3>
    <p>The results of SDS-PAGE gel electrophoresis and Western blot indicate that the GLP-1 recombinant protein is indeed expressed in the supernatant of the engineered bacteria, confirming that it has been successfully secreted into the extracellular space.</p>

    <h3>Study</h3>
    <p>Based on the above findings, we will conduct functional validation of the target gene in the upcoming iterations.

    </p>


  </div>
</div>

<div class="row mt-4">
  <div class="col">
    <h2>3.1. First Iteration - Gene Function Validation Engineering</h2>
    <hr>

    <h3>Design</h3>
    <p>In this iteration, our primary goal is to assess the function of the BCoAT gene, specifically focusing on the rate and levels of short-chain fatty acids (SCFAs) production, primarily acetate, propionate, and butyrate.</p>

    <h3>Build</h3>
    <p>We used high-performance liquid chromatography (HPLC) to measure the expression of three SCFAs—acetate, propionate, and butyrate—produced by engineered bacteria under time-series conditions, as an indicator of BCoAT gene functionality.

    </p>

    <h3>Test</h3>
    <p>The results indicate that, compared to the chassis bacteria, the engineered bacteria were able to produce all three short-chain fatty acids (SCFAs). This confirms that the BCoAT gene is functioning normally in the engineered strains. Time-series analysis revealed that, 12 hours after inoculation into the new medium, the concentrations of the three SCFAs gradually increased, which corresponds to the transition of the engineered bacteria from the logarithmic growth phase to the stationary phase.

    </p>

    <h3>Study</h3>
    <p>Although we have validated the ability of the engineered bacteria to produce SCFAs in this section, we have not confirmed the functional relationship between SCFAs and glucose regulation (although this has been reported in the literature, it does not prove that the engineered bacteria have the same effect). We plan to conduct in vitro cell experiments in the next phase to verify this effect.

    </p>
  </div>
</div>

<div class="row mt-4">
  <div class="col">
    <h2>3.2. Second Iteration - Gene Function Validation Engineering</h2>
    <hr>


    
    <h3>Design</h3>
    <p>In this iteration, we will verify the functionality of the GLP-1 recombinant protein, primarily its role in glucose regulation through GLP-1 release.

    </p>

    <h3>Build</h3>
    <p>We established a gradient of glucose concentrations in the culture medium, as well as a time series at the highest glucose concentration (simulating a hyperglycemic environment) to cultivate the engineered bacteria. Afterward, we collected the supernatant from the culture by centrifugation and used the ELISA method to measure the concentration of GLP-1 in the supernatant, inferring the results of its release.

    </p>

    <h3>Test</h3>
    <p>The results indicate that, under the same cultivation time conditions, the concentration of GLP-1 in the supernatant increased with the rising glucose content in the culture medium. In the time series measurements at the highest glucose concentration (11 mmol/L), the GLP-1 concentration gradually rose over the 12-hour period and then leveled off.

    </p>

    <h3>Study</h3>
    <p>Although the designed GLP-1 recombinant protein achieved the expected functionality, we did not conduct more detailed tests on the individual components of the recombinant protein, which is necessary for future work. Additionally, we need to consider whether the GLP-1 recombinant protein can exert biological value in regulating blood glucose levels, which will be validated in future cell experiments.


    </p>
  </div>
</div>


  </div>

  <div class="home" style="position:relative;top:-3rem;overflow-x: hidden;">
    
  </div>

  <footer class="pt-5 pb-5 footer py-5 mt-5 bg-dark text-white">
  <div class="container">
    <div class="row mb-4">
      <div class="col-lg-8 col-xs-12">
        <h4 class="mb-3">About Us</h4>
        <div class="row">
          <div class="col-lg-6 col-xs-12 text-left">
            <img src="static/footer-logo/logo-lzu.png" height="100px" class="mr-4">
            <a>Lanzhou University</a>
          </div>
        </div>
        <div class="row">
          <div class="col-lg-6 col-xs-12 text-left">
            <img src="static/footer-logo/footer.png" height="100px" class="mr-4">
          </div>
        </div>
        <div class="row">
          <div class="col-lg-6 col-xs-12 text-left">
            <img src="static/footer-logo/footerlogo2.png" height="100px" class="mr-4">
          </div>
        </div>
      </div>

      <div class="col-lg-4 col-xs-12">
        <h4 class="mt-lg-0 mt-sm-4 mb-3">Contact</h4>
        <p>No. 222, Tianshui South Road, Chengguan District, Lanzhou City, Gansu Province,730000, P. R. China.</p>
        <p><b>E-mail:</b> <a href="mailto:11@lzu.edu.cn">susu2969@foxmail.com</a></p>
      </div>
    </div>
    <hr>
    <div class="md-copyright__highlight">
      Copyright &copy; 2024 iDEC. CC-BY-4.0 | The International Directed Evolution Competition (iDEC) is a Scottish Charitable Incorporate Organisation (SCIO), registered in Scotland, with charity number：SC050826
    </div>
  </div>
</footer>


  <!-- Bootstrap Bundle with Popper -->
  <script src="static/bootstrap.bundle.min.js"></script>
</body>
</html>